modules/openy_gc_auth/modules/openy_gc_auth_reclique/src/EventSubscriber/GCAuthReCliqueUserLoginSubscriber.php

```diff
@@ -66,14 +66,24 @@ public function onUserLogin(GCUserLoginEvent $event) {
             }
           }
           $user_membership = $event->extraData['member']['PackageName'];
```

Comment:
Seems like this variable is not used and can be removed
`$user_membership = $event->extraData['member']['PackageName'];`

Reply:
@TomScarboroughYMCA ⬆️

Reply:
Thanks - good catch